old_code/compute_distances.py

```diff
@@ -0,0 +1,124 @@
+"""  Data Visualization Sarah Barden and Alisha Pegan
```

Comment:
You don't need to have an "old code" folder because github remembers all of your old code.